os/tools/compression/src/compress.c

```diff
@@ -65,6 +65,7 @@ static void compress_file(int block_size, int type, char *in_file, char *out_fil
 #if CONFIG_COMPRESSION_TYPE == LZMA
 	long unsigned int propsSize = LZMA_PROPS_SIZE;
 #endif
+	int exit_code = 2;
```

Comment:
Is it possible to show the meaning of "2" ?

Reply:
Then how about defining them at public header to know the reason easily?

```
xxx.h
#define COMP_SUCCESS          0
#define COMP_INVALID_TYPE  1
#define COMP_FAILURE            2
#define COMP_NOT_SUPPORT 3
#define COMP_INVALID_INPUT 4

xxx.c
exit(COMP_SUCCESS);
exit(COMP_FAILURE);
...
```

Reply:
Done.